pkgs/top-level/perl-packages.nix

```diff
@@ -12665,16 +12665,23 @@ with self;
       hash = "sha256-u+rO2ZXX2NEM/FGjpaZtpBzrK8BP7cq1DhDmMA6AHG4=";
     };
     nativeBuildInputs = [ buildPackages.pkg-config ];
```

Comment:
could try changing this to `depsBuildBuild = [ pkg-config ];` if you're interested in cleaning up the dependencies in general.